.buildkite/scripts/steps/ecp-internal-release.sh

```diff
@@ -15,12 +15,11 @@
 # this material is strictly forbidden unless prior written
```

Comment:
wonder if this script should be owned by a specific team in the CODEOWNERS, so any changes need their approval and we, robots, don't need to be in the review process.